workspaces/ui/src/components/diff/v2/AddUrlModal.js

```diff
@@ -197,7 +204,12 @@ export const NewUrlModal = withRfcContext((props) => {
             />
           </DialogContent>
           <DialogActions>
-            <Button onClick={() => setNaming(false)}>Back</Button>
+            <Button
+              disabled={Boolean(knownPathId)}
```

Comment:
@LouManglass -- this is the main thing that changed. The rest is noisy formatting